<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Panel</class>
 <widget class="QWidget" name="Panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1557</width>
    <height>287</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="CircularGauge" name="ground_speed_display" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>250</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Click and drag here</string>
     </property>
     <property name="whatsThis">
      <string>Circular Gauge Widget.  </string>
     </property>
     <property name="label_font" stdset="0">
      <font>
       <family>Rockwell</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="value_font" stdset="0">
      <font>
       <family>Rockwell</family>
       <pointsize>60</pointsize>
      </font>
     </property>
     <property name="label" stdset="0">
      <string>KM/H</string>
     </property>
     <property name="value" stdset="0">
      <number>125</number>
     </property>
     <property name="max_value" stdset="0">
      <number>250</number>
     </property>
     <property name="bar_background" stdset="0">
      <brush brushstyle="SolidPattern">
       <color alpha="30">
        <red>77</red>
        <green>104</green>
        <blue>112</blue>
       </color>
      </brush>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>100</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="gridWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0" columnstretch="0,0,0,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <property name="spacing">
         <number>5</number>
        </property>
        <item row="2" column="2">
         <widget class="QFrame" name="frame_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background: rgba(30, 98, 87, 10)</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="IconData" name="engine_temperature_display" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Click and drag here</string>
             </property>
             <property name="whatsThis">
              <string>Icon Data Widget.  </string>
             </property>
             <property name="value" stdset="0">
              <number>198</number>
             </property>
             <property name="label" stdset="0">
              <string>°C</string>
             </property>
             <property name="value_font" stdset="0">
              <font>
               <family>Rockwell</family>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="label_font" stdset="0">
              <font>
               <family>Rockwell</family>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="scale" stdset="0">
              <double>0.800000000000000</double>
             </property>
             <property name="icon" stdset="0">
              <iconset resource="../resources/icons.qrc">
               <normaloff>:/thermometer/assets/thermometer/Car_Engine_-_Dashboard_Lights_110-512.png</normaloff>:/thermometer/assets/thermometer/Car_Engine_-_Dashboard_Lights_110-512.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QFrame" name="frame_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background: rgba(30, 98, 87, 10)</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="IconData" name="throttle_display" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Click and drag here</string>
             </property>
             <property name="whatsThis">
              <string>Icon Data Widget.  </string>
             </property>
             <property name="value" stdset="0">
              <number>60</number>
             </property>
             <property name="label" stdset="0">
              <string>%</string>
             </property>
             <property name="value_font" stdset="0">
              <font>
               <family>Rockwell</family>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="label_font" stdset="0">
              <font>
               <family>Rockwell</family>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="scale" stdset="0">
              <double>0.800000000000000</double>
             </property>
             <property name="icon" stdset="0">
              <iconset resource="../resources/icons.qrc">
               <normaloff>:/throttle/assets/throttle/2303987-200.png</normaloff>:/throttle/assets/throttle/2303987-200.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QFrame" name="frame_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background: rgba(30, 98, 87, 10)</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="IconData" name="air_temperature_display" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Click and drag here</string>
             </property>
             <property name="whatsThis">
              <string>Icon Data Widget.  </string>
             </property>
             <property name="value" stdset="0">
              <number>35</number>
             </property>
             <property name="label" stdset="0">
              <string>°C</string>
             </property>
             <property name="value_font" stdset="0">
              <font>
               <family>Rockwell</family>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="label_font" stdset="0">
              <font>
               <family>Rockwell</family>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="scale" stdset="0">
              <double>0.800000000000000</double>
             </property>
             <property name="icon" stdset="0">
              <iconset resource="../resources/icons.qrc">
               <normaloff>:/thermometer/assets/thermometer/120779.png</normaloff>:/thermometer/assets/thermometer/120779.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QFrame" name="frame_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background: rgba(30, 98, 87, 10)</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="IconData" name="lambda_display" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Click and drag here</string>
             </property>
             <property name="whatsThis">
              <string>Icon Data Widget.  </string>
             </property>
             <property name="value" stdset="0">
              <number>12</number>
             </property>
             <property name="label" stdset="0">
              <string>V</string>
             </property>
             <property name="value_font" stdset="0">
              <font>
               <family>Rockwell</family>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="label_font" stdset="0">
              <font>
               <family>Rockwell</family>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="scale" stdset="0">
              <double>0.800000000000000</double>
             </property>
             <property name="icon" stdset="0">
              <iconset resource="../resources/icons.qrc">
               <normaloff>:/lambda/assets/lambda/lambda-97846_960_720.png</normaloff>:/lambda/assets/lambda/lambda-97846_960_720.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QFrame" name="frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background: rgba(30, 98, 87, 10)</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="IconData" name="battery_display" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Click and drag here</string>
             </property>
             <property name="whatsThis">
              <string>Icon Data Widget.  </string>
             </property>
             <property name="value" stdset="0">
              <number>12</number>
             </property>
             <property name="label" stdset="0">
              <string>V</string>
             </property>
             <property name="value_font" stdset="0">
              <font>
               <family>Rockwell</family>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="label_font" stdset="0">
              <font>
               <family>Rockwell</family>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="scale" stdset="0">
              <double>0.800000000000000</double>
             </property>
             <property name="icon" stdset="0">
              <iconset resource="../resources/icons.qrc">
               <normaloff>:/battery/assets/battery/car+battery+icon-1320184285905906096.png</normaloff>:/battery/assets/battery/car+battery+icon-1320184285905906096.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QFrame" name="frame_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background: rgba(30, 98, 87, 10)</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="IconData" name="oil_pressure_display" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Click and drag here</string>
             </property>
             <property name="whatsThis">
              <string>Icon Data Widget.  </string>
             </property>
             <property name="value" stdset="0">
              <number>2</number>
             </property>
             <property name="label" stdset="0">
              <string>kPa</string>
             </property>
             <property name="value_font" stdset="0">
              <font>
               <family>Rockwell</family>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="label_font" stdset="0">
              <font>
               <family>Rockwell</family>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="scale" stdset="0">
              <double>0.800000000000000</double>
             </property>
             <property name="icon" stdset="0">
              <iconset resource="../resources/icons.qrc">
               <normaloff>:/oil/assets/oil/17696-200.png</normaloff>:/oil/assets/oil/17696-200.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QFrame" name="frame_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background: rgba(30, 98, 87, 10)</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="IconData" name="fuel_used_display" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Click and drag here</string>
             </property>
             <property name="whatsThis">
              <string>Icon Data Widget.  </string>
             </property>
             <property name="value" stdset="0">
              <number>9</number>
             </property>
             <property name="label" stdset="0">
              <string>L</string>
             </property>
             <property name="value_font" stdset="0">
              <font>
               <family>Rockwell</family>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="label_font" stdset="0">
              <font>
               <family>Rockwell</family>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="scale" stdset="0">
              <double>0.800000000000000</double>
             </property>
             <property name="icon" stdset="0">
              <iconset resource="../resources/icons.qrc">
               <normaloff>:/fuel/assets/fuel/fuel.png</normaloff>:/fuel/assets/fuel/fuel.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QFrame" name="frame_15">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background: rgba(30, 98, 87, 10)</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_16">
           <item>
            <widget class="IconData" name="ignition_advance_display" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Click and drag here</string>
             </property>
             <property name="whatsThis">
              <string>Icon Data Widget.  </string>
             </property>
             <property name="value" stdset="0">
              <number>9</number>
             </property>
             <property name="label" stdset="0">
              <string>°</string>
             </property>
             <property name="value_font" stdset="0">
              <font>
               <family>Rockwell</family>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="label_font" stdset="0">
              <font>
               <family>Rockwell</family>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="scale" stdset="0">
              <double>0.800000000000000</double>
             </property>
             <property name="icon" stdset="0">
              <iconset resource="../resources/icons.qrc">
               <normaloff>:/ignition/assets/ignition/50425-200.png</normaloff>:/ignition/assets/ignition/50425-200.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>100</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CircularGauge" name="rpm_display" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>250</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Click and drag here</string>
     </property>
     <property name="whatsThis">
      <string>Circular Gauge Widget.  </string>
     </property>
     <property name="label_font" stdset="0">
      <font>
       <family>Rockwell</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="value_font" stdset="0">
      <font>
       <family>Rockwell</family>
       <pointsize>60</pointsize>
      </font>
     </property>
     <property name="value" stdset="0">
      <number>10000</number>
     </property>
     <property name="min_value" stdset="0">
      <number>0</number>
     </property>
     <property name="max_value" stdset="0">
      <number>20000</number>
     </property>
     <property name="bar_background" stdset="0">
      <brush brushstyle="SolidPattern">
       <color alpha="30">
        <red>77</red>
        <green>104</green>
        <blue>112</blue>
       </color>
      </brush>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CircularGauge</class>
   <extends>QWidget</extends>
   <header>src/widgets/gaugewidget</header>
  </customwidget>
  <customwidget>
   <class>IconData</class>
   <extends>QWidget</extends>
   <header>src/widgets/icondatawidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
